lib/client/client.js

```diff
@@ -829,7 +840,7 @@ Client.prototype.connect = function connect () {
       socket.once('connect', onConnect)
     }
     socket.once('error', onResult)
-    initSocket()
+    initSocket(self.url)
```

Comment:
This should be `initSocket(self.urls[self._nextServer])` and `self.url` should be `undefined` until the server connection is complete and we know which url we are actually connected to. Probably could be done in the `connectSocket` [callback](https://github.com/wision/node-ldapjs/blob/multi-servers/lib/client/client.js#L1009)